roave-bc-check.yaml

```diff
@@ -1,5 +1,5 @@
 parameters:
     ignoreErrors:
-        - '#\[BC\] CHANGED: Property Faker\\Provider\\[a-zA-Z]+::\$[a-zA-Z]+ changed default value#'
-        - '#\[BC\] CHANGED: Property Faker\\Provider\\[a-z_A-Z]+\\[a-zA-Z]+::\$[a-zA-Z]+ changed default value#'
+        - '#\[BC\] CHANGED: Property Faker\\Provider\\[a-zA-Z]+::\$[1-9a-zA-Z]+ changed default value#'
```

Comment:
Why not go from 0-9 here?

Reply:
There is no usecase for such class at this point so thats the makn reason.

Reply:
@bramceulemans can you review again?